ui/app/src/utils/i18n-legacy.ts

```diff
@@ -135,6 +135,17 @@ const contentTypesMessages = defineMessages({
 
 });
 
+const commonMessages = defineMessages({
+  confirm: {
+    id: 'commonMessages.confirm',
+    defaultMessage: 'Confirm'
+  },
+  cancel: {
+    id: 'commonMessages.cancel',
+    defaultMessage: 'Cancel'
+  }
+});
+
```

Comment:
Rename message bundle to words as per code autocompletion PR. 

You may equalize these two PR messages to avoid conflicts (i.e. include the strings you're using in both PRs in both PRs)